ganttproject/src/biz/ganttproject/impex/csv/CommonWriter.java

```diff
@@ -0,0 +1,15 @@
+package biz.ganttproject.impex.csv;
+
+import java.io.Closeable;
+import java.io.Flushable;
+import java.io.IOException;
+
+/**
+ * @author akurutin on 04.04.2017.
+ */
+public interface CommonWriter extends Flushable, Closeable {
+    public void print(Object value) throws IOException;
```

Comment:
I thought we always print `String`s in the exporter code. If there is any place where we output something other than string then probably we can just fix that place and use `String` here always?